### DISCUSSION TITLE
Update README advanced section with other environment vars

### DISCUSSION THREAD

**Discussion by ToshY** [louislam/dockge#8] [Feature Request]
Looking at [`dockge-server.ts`](https://github.com/louislam/dockge/blob/6dc998bedf24bf68b17d6a2d55e68361c26e17f8/backend/dockge-server.ts#L128) it is possible to set more environment variables than denoted in the current [advanced section](https://github.com/louislam/dockge?tab=readme-ov-file#advanced). Would be nice if the advanced section can be updated to include these environment variables.

**Comment by louislam** [comment]
Yes, I will list them in the wiki later.
